mlir/lib/Transform/AIRDmaToChannel.cpp

```diff
@@ -112,106 +112,6 @@ static SmallVector<Value, 1> getLoopTokens(scf::ParallelOp loop) {
   return output;
 }
 
-static void replaceAllUsesOfInductionVarsWith(scf::ForOp old_loop,
-                                              scf::ForOp new_loop) {
-  replaceAllUsesInRegionWith(old_loop.getInductionVar(),
-                             new_loop.getInductionVar(), new_loop.getRegion());
-}
-
-static void replaceAllUsesOfInductionVarsWith(scf::ParallelOp old_loop,
-                                              scf::ParallelOp new_loop) {
-  for (unsigned i = 0; i < old_loop.getInductionVars().size(); i++) {
-    replaceAllUsesInRegionWith(old_loop.getInductionVars()[i],
-                               new_loop.getInductionVars()[i],
-                               new_loop.getRegion());
-  }
-}
-
-static void replaceAllUsesOfLoopTokensWith(scf::ForOp old_loop,
-                                           scf::ForOp new_loop) {
-  if (old_loop.getRegionIterArgs().size()) {
-    for (unsigned i = 0; i < old_loop.getRegionIterArgs().size(); i++) {
-      replaceAllUsesInRegionWith(old_loop.getRegionIterArgs()[i],
-                                 new_loop.getRegionIterArgs()[i],
-                                 new_loop.getRegion());
-    }
-  }
-}
-
-static void replaceAllUsesOfLoopTokensWith(scf::ParallelOp old_loop,
-                                           scf::ParallelOp new_loop) {
-  if (old_loop.getInitVals().size()) {
-    for (unsigned i = 0; i < old_loop.getInitVals().size(); i++) {
-      replaceAllUsesInRegionWith(old_loop.getInitVals()[i],
-                                 new_loop.getInitVals()[i],
-                                 new_loop.getRegion());
```

Comment:
This one was already redundant with the remap done when `new_loop` was created.

Reply:
Agreed. Makes sense to me.